packages/console/src/pages/ApplicationDetails/components/AddAnotherButton/index.tsx

```diff
@@ -0,0 +1,23 @@
+import classNames from 'classnames';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import * as textButtonStyles from '@/components/TextButton/index.module.scss';
+
+import * as styles from './index.module.scss';
+
+type Porps = {
+  onClick: () => void;
+};
+
+const AddAnotherButton = ({ onClick }: Porps) => {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+
+  return (
+    <div className={classNames(textButtonStyles.button, styles.addButton)} onClick={onClick}>
+      {`+ ${t('application_details.add_another')}`}
```

Comment:
nit: could ${t('application_details.add_another')} be undefined?  if empty should not show '+' as well

Reply:
Also, should it be a general link button or text button that takes a string as its child instead of an explicit AddAnotherButton component?

Reply:
I agree, should be `Button type="plain"`. But probably need to wait for the button component refactor to support "icon + title" as children.

Reply:
I am gonna make a "IconButton" component LOG-1890

Reply:
Rebased